justfile

```
@@ -54,7 +54,7 @@ download-deps:
 	wget https://github.com/CosmWasm/cw-plus/releases/latest/download/cw20_base.wasm -O artifacts/cw20_base.wasm
 # TODO?: test dao-contracts
 
-all: build test check schema gen optimize checksum
+all: build test lint schema gen optimize checksum
```

Let's have lint before builds and tests!
```suggestion
all: lint build test schema gen optimize checksum
```

---

this one is fixed in one of PRs))